modules/order/src/AdjustableInterface.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace Drupal\commerce_order;
+
+/**
+ * Defines an interface for objects that interact with Adjustments.
+ *
+ * @todo Provide a trait? But that'd assume the field is always 'adjustments'
```

Comment:
I went to try this. The awkwardness isn't the field, since we could provide a method which provides the definition. But more so needing to calculate a total in the set methods. Felt weird to provide an `abstract protect function calculateTotals()` in the trait.